lib/modules/apostrophe-groups/index.js

```diff
@@ -64,6 +64,9 @@ module.exports = {
     options.removeFilters = [ 'published' ]
       .concat(options.removeFilters || []);
 
+    options.removeColumns = (options.defaultRemoveColumns || [ 'published' ])
```

Comment:
I don't know if this one needs the `options.defaultRemoveColumns` option since there isn't a similar option for `removeFilters`. None of these seems to be documented either, so it seems safe to make this simpler.